### DISCUSSION TITLE
A star cluster in the Milky Way appears to be as old as the [ΛCDM] universe

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#72] [Astrophysics and Physics]
In [Science News](https://www.sciencenews.org/article/star-cluster-milky-way-old-universe), original paper [arXiv:2306.02180](https://arxiv.org/abs/2306.02180)

Or is it that the ΛCDM universe appears as young as a star cluster in the Milky Way?

![M92 is too old](https://github.com/a-cosmology-group/acg/assets/16748402/fb829722-7e2c-4f9b-b5f1-a6977591cd70)

**Comment by budrap00** [comment]
You would think death by a thousand cuts... but we are dealing with the LCDM zombie here. It's already dead yet lumbers on.

**Comment by khuramonline** [comment]
> “We often think about it as, Moses came down from Mount Sinai with ‘13.8 billion years’ written on some tablets or something, but it’s not quite like that,” he says. “If one takes the Hubble tension seriously, then one also has to say we don’t know the age of the universe that well.”

They are trying to make ground for the next tweak in their expanding universe theory and the doctrine of 13.8 BY age of the Universe.

But their all other "proofs" depend on exact 13.8 BY age of the Universe. **CMB's exact right as "predicted"** temperature comes from exact age of 13.8 BY age of Universe. Even MOND calculations also have incorporated age of universe 13.8 BY.

They will also need to modify their doctrine of dark ages because through formulas there is full stop for visible galaxies up to the distance of 13.6 BLYs. If let's say new "age" of universe is set at 14.2 BYs and dark age period is still to be regarded as 300 MYs then they will need to change formula for telling us the distances of galaxies. For the galaxies having current distance of 13.5 BLYs, they will reset their distance to be 13.9 BLYs. All this will depend on formulas.

And perhaps they will also change the formulas for calculating the temperature of CMB to get same current value but with different age of universe to again tell us that CMB is the same thing that was "predicted" by their theory.. And off course it will be done with the precision of 14 decimal digits.

**Comment by HanDeBruijn** [comment]
Perhaps an (Orbital) age twice as large as the Hubble time would help? That's what Arp and Narlikar have been proposing.

**Comment by khuramonline** [comment]
This futile attempt of exact quantification of cosmological distances, temperatures and age of universe etc. clearly indicates that Cosmology belongs to the domain of Philosophy.

Problem is more complicated because Theoretical Physics itself is a form of philosophy but it is philosophy under the guise of science. Theoretical Physics is a form of Rationalism Philosophy with mathematical touch of (futile types of) exact quantifications for the whole of universe.
